internal/session/handle.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 func (s *Session) handleOther(ctx context.Context, tag string, cmd *proto.Command, profiler profiling.CmdProfiler) chan response.Response {
-	ch := make(chan response.Response)
+	ch := make(chan response.Response, 100)
```

Comment:
why buffer this channel? and how'd you arrive at 100?

Reply:
If you don't buffer the channel, you can only send one item and then you need to wait for that one item to be read. If it takes some time to send the reply from gluon to the client, the command code can't proceed until gluon fetches the next response. Buffering now allows the command to execute and release all the locks sooner as it can queue up N response items without waiting for them to be sent.

The number 100 is arbitrary, I was considering large fetch request for this case.

Reply:
Good point, nice change.

In that case, i'd like the 100 to be defined as a package constant, potentially even a variable such that it can be configured (in future, not in this MR) by a gluon init option.